client/blocks/reader-full-post/style.scss

```diff
@@ -103,22 +81,11 @@
 
 .reader-full-post__sidebar {
 	width: 220px;
-	position: fixed;
 	text-align: center;
+	margin-top: 55px;
 
-	@include breakpoint-deprecated( ">1280px" ) {
-		left: calc(50% - 490px);
-	}
-
-	@include breakpoint-deprecated( "1040px-1280px" ) {
-		left: calc(50% - 480px);
-	}
-
-	@include breakpoint-deprecated( "660px-1040px" ) {
-		left: 20px;
-	}
-
-	@include breakpoint-deprecated( "<660px" ) {
+	// hide the sidebar when it cannot fit to the side of the content.
+	@media (max-width: 1298px) {
```

Comment:
Re: gutter padding. If we adjust that we will need to tweak this value similarly. I may look into using variables to connect these values better.